src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={todo.completed
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo'}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    { todo.user ? <UserInfo user={todo.user} /> : '' }
```

Comment:
The ternary operator is redundant here 
```suggestion
    { user && <UserInfo user={todo.user} /> }
```